package.json

```diff
@@ -38,11 +38,14 @@
     "test:p2p:watch": "mocha --timeout 10000 -r ts-node/register test/p2p/*  --watch --watch-extensions ts",
     "test:crypto": "mocha --timeout 10000 -r ts-node/register test/crypto/*",
     "test:sim": "(npm run test:sim:build && npm run test:sim:run)",
-    "test:sim:build": "(npm run compile:dev && mkdir -p test/simulation/docker-xud/xud && cp -a dist test/simulation/docker-xud/xud && cp -a bin test/simulation/docker-xud/xud && cp package.json test/simulation/docker-xud/xud && cd test/simulation && ./docker-build.sh)",
+    "test:sim:build": "(npm run test:sim:update-xud && cd test/simulation && ./docker-build.sh)",
     "test:sim:build:test": "(cd test/simulation && ./docker-build.sh test)",
+    "test:sim:build:xud": "(npm run test:sim:update-xud && cd test/simulation && ./docker-build.sh xud)",
```

Comment:
When should we run this script? It's not used by other scripts and I don't see it mentioned in the readme.

Reply:
Like you said, this is script is needed whenever non-test code is changed. 
It is included inside the `test:sim:build` procedure.